Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -404,107 +352,72 @@ void CGPathAddArc(CGMutablePathRef path,
     }
 
     // Call the internal function for breaking the arcs down into smaller segments
-    _CGPathAddArc(path, m, x, y, radius, startAngle, endAngle, clockwise);
+    _CGPathAddArc(path, transform, x, y, radius, startAngle, endAngle, clockwise);
 }
 
 /**
  @Status Interoperable
 */
-void CGPathMoveToPoint(CGMutablePathRef path, const CGAffineTransform* m, float x, float y) {
-    if (path == NULL) {
-        return;
-    }
-
-    if (m) {
-        CGPoint pt;
-
-        pt.x = x;
-        pt.y = y;
-
-        pt = CGPointApplyAffineTransform(pt, *m);
-
-        x = pt.x;
-        y = pt.y;
+void CGPathMoveToPoint(CGMutablePathRef path, const CGAffineTransform* transform, float x, float y) {
```

Comment:
Should I file an issue?

---
In reply to: [83963132](https://github.com/Microsoft/WinObjC/pull/1138#discussion_r83963132) [](ancestors = 83963132,83946485,83932143)